docs/rtk-query/usage/code-generation.mdx

```diff
@@ -42,6 +44,8 @@ export const emptySplitApi = createApi({
 Generate a config file (json, js or ts) with contents like
 
 ```ts no-transpile
+// openapi-config.ts
```

Comment:
```suggestion
```ts no-transpile title="openapi-config.ts"
```

Reply:
Hey, great input! Could you please format it like that? Then it should show up like in the [Quick Start](https://redux-toolkit.js.org/tutorials/quick-start) code examples (look for `index.js`)

Reply:
ah fantastic, will do!